gui.lua

```diff
@@ -1791,6 +1791,29 @@ function GUI:Init()
                     StaticPopup_Show("RAIDLEDGER_DELETE_ITEM")
                 else
                     ChatEdit_InsertLink(entry["detail"]["item"])
+
+                    if IsShiftKeyDown() then

```

Comment:
shift/alt + up / down seems more friendly and easier to understand

Reply:
because I don't know how to target&focus a row in the table...hence I created this way to move an item...